client/homebrew/pages/basePages/uiPage/uiPage.jsx

```diff
@@ -2,7 +2,7 @@ require('./uiPage.less');
 const React = require('react');
 const createClass = require('create-react-class');
 
-const Nav = require('naturalcrit/nav/nav.jsx');
+const Nav = require('client/homebrew/navbar/nav.jsx');
 const Navbar = require('../../../navbar/navbar.jsx');
 const NewBrewItem = require('../../../navbar/newbrew.navitem.jsx');
 const HelpNavItem = require('../../../navbar/help.navitem.jsx');
```

Comment:
Since all of these are now imported from the same navbar folder, it's a little confusing to see some paths from `client/homebrew/` and others from `../../../`. Would you mind making them all the `navbar/` folder imports consistent with each other?

Reply:
okay, i just did that across references to the navbar stuff across all files.